PyExpLabSys/drivers/srs_sr830.py

```diff
@@ -0,0 +1,98 @@
+""" Simple driver for Keithley SRS SR830 Lock-In Amplifier """
+import time
+from PyExpLabSys.drivers.scpi import SCPI
+
+
+class SR830(SCPI):
+    """
+    Simple driver for SRS SR830
+    """
+
+    def __init__(self, interface, gpib_address=None):
+        if interface == 'gpib':
+            SCPI.__init__(self, interface=interface,
+                          gpib_address=gpib_address, line_ending='\n')
+            self.comm_dev.clear()
+            time.sleep(0.2)
+
+    def use_external_reference(self, trigger: str):
+        """
+        Configures the reference frequency to be externally generated,
+        the trigger can take the values sine, rising and falling.
+        """
+        if trigger == 'sine':
+            param = 0
+        elif trigger == 'rising':
+            param = 1
+        elif trigger == 'falling':
+            param = 2
+        else:
+            return False
```

Comment:
I'd suggest raising ValueError with an explanation of the options

Reply:
Might be theoretically correct, but a bit of mess to handle currently. I have made comment to future me.